presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -28,6 +28,7 @@
 import javax.annotation.concurrent.GuardedBy;
 import javax.annotation.concurrent.ThreadSafe;
 
+import java.util.Comparator;
```

Comment:
#6468 only mentions query timeouts not hard memory limits. Please create separate issues if you wish.